python/src/usecases/stiffened_panel.py

```diff
@@ -135,7 +144,33 @@ def __init__(self):
         self.p = ot.Uniform(0.03762, 0.03838)
         self.p.setDescription(["p (m)"])
 
+        # correlation matrix
+        self.correlation_matrix = ot.CorrelationMatrix(self.dim)
+        self.correlation_matrix[4, 5] = -0.8
+        self.copula = ot.NormalCopula(
+            ot.NormalCopula.GetCorrelationFromSpearmanCorrelation(
+                self.correlation_matrix
+            )
+        )
+
         self.distribution = ot.JointDistribution(
+            [
+                self.youngModulus,
+                self.nu,
+                self.h_c,
+                self.ell,
+                self.f_1,
+                self.f_2,
+                self.t,
+                self.a,
+                self.b_0,
+                self.p,
+            ],
```

Comment:
lets not repeat the var list for the two distributions

Reply:
I don't get it. 
Do you want something like 
```python
varList = [self.E, ....]
distribution = ot.JointDistribution(varList, copula)
independentDistribution = ot.JointDistribution(varList)
```
instead?

Reply:
yes that works

Reply:
OK,  done.